src/domain/community/community/community.resolver.mutations.ts

```diff
@@ -424,7 +455,7 @@ export class CommunityResolverMutations {
     @CurrentUser() agentInfo: AgentInfo,
     @Args('invitationData')
     invitationData: CreateInvitationExternalUserOnCommunityInput
-  ): Promise<IInvitationExternal> {
+  ): Promise<IInvitation | IInvitationExternal> {
```

Comment:
It bugs me that the mutation is named `inviteExternalUser...` but you are allowed to use it on anyone.

Reply:
Yes! That's even better.